ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksTable.tsx

```diff
@@ -106,91 +107,126 @@ function ProfileChecksTable({
                         message: 'Clear all filters and try again',
                     }}
                     renderer={({ data }) => (
-                        <Tbody>
-                            {data.map((check) => {
-                                const { checkName, rationale, checkStats } = check;
+                        <>
+                            {data.map((check, rowIndex) => {
+                                const { checkName, rationale, checkStats, controls } = check;
                                 const { passCount, failCount, otherCount, totalCount } =
                                     getStatusCounts(checkStats);
                                 const passPercentage = calculateCompliancePercentage(
                                     passCount,
                                     totalCount
                                 );
                                 const progressBarId = `progress-bar-${checkName}`;
+                                const isRowExpanded = expandedRows.includes(rowIndex);
```

Comment:
Because you mentioned set type earlier today, I was curious to find if any similar examples use set.

Not for this contribution, but replace array with set might be in scope for 4.5I sprint.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx#L208

Reply:
Good callout